mysql/time.go

```diff
@@ -215,17 +215,26 @@ func (t *Time) UnmarshalInLocation(b []byte, loc *time.Location) error {
 }
 
 const numberFormat = "20060102150405"
+const dateFormat = "20060102" // see issue #1046
 
 // ToNumber returns a formatted number.
 // e.g,
 // 2012-12-12T10:10:10 -> 20121212101010
 // 2012-12-12T10:10:10.123456 -> 20121212101010.123456
```

Comment:
Add a new example in the comment to make it consist with the new implementation.